src/twyn/core/config_handler.py

```diff
@@ -59,32 +152,35 @@ def _get_toml_file_pointer(self) -> Path:
 
         return fp
 
-    def _write_toml(self) -> None:
+    def _write_toml(self, toml: TOMLDocument) -> None:
         with open(self._get_toml_file_pointer(), "w") as f:
-            f.write(dumps(self._toml))
-
-    def _get_toml_as_dict(self) -> dict[str, Any]:
-        """Read TOML into a dictionary."""
-        try:
-            fp = self._get_toml_file_pointer()
-        except FileNotFoundError:
-            if not self._enforce_file and self._file_path == DEFAULT_PROJECT_TOML_FILE:
-                return {}
-            raise
-
-        with open(fp, "r") as f:
-            content = parse(f.read())
-        return parse(dumps(content))
-
-    def _create_allowlist_in_toml_if_not_exists(self) -> None:
-        try:
-            isinstance(self._toml["tool"]["twyn"]["allowlist"], list)
-        except KeyError:
-            if "tool" not in self._toml:
-                self._toml["tool"] = {}
-
-            if "twyn" not in self._toml["tool"]:
-                self._toml["tool"]["twyn"] = {}
-
-            if "allowlist" not in self._toml["tool"]["twyn"]:
-                self._toml["tool"]["twyn"]["allowlist"] = []
+            try:
+                f.write(dumps(toml))
+            except Exception:
+                logger.exception("Error writing toml file")
+                raise TOMLError(f"Error writing toml to {self._file_path}") from None
+
+
+def _get_logging_level(
```

Comment:
should this be inside the class?

Reply:
Since it does not use `self` at all, I just put it here.